File: rb/utility/make_context.rb
# ZefoyTiktokBot SDK utility: make_context
require_relative '../core/context'
module ZefoyTiktokBotUtilities
  MakeContext = ->(ctxmap, basectx) {
    ZefoyTiktokBotContext.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// ZefoyTiktokBot SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class ZefoyTiktokBotMakeContext
{
    public static function call(array $ctxmap, ?ZefoyTiktokBotContext $basectx): ZefoyTiktokBotContext
    {
        return new ZefoyTiktokBotContext($ctxmap, $basectx);
    }
}
